docs/sdks/uid2-sdk-ref-ios.md

```diff
@@ -120,15 +135,21 @@ This section includes the functions and variables that are part of the UID2Manag
 ### Functions
 
 The following functions are available as part of the UID2Manager API:
+- [generateIdentity()](#generateidentity)
 - [setIdentity()](#setidentity)
 - [resetIdentity()](#resetidentity)
 - [refreshIdentity()](#refreshidentity)
 - [getAdvertisingToken()](#getadvertisingtoken)
 - [setAutomaticRefreshEnabled()](#setautomaticrefreshenabled)
 
+#### generateIdentity()
+
+Generate a UID2 Identity using a <Link href="../ref-info/glossary-uid#gl-dii">Directly identifying information (DII)</Link>. Refer to
+[Client-Server Integration Guide for Mobile](https://ttdcorp-my.sharepoint.com/:w:/g/personal/sunny_wu_thetradedesk_com/EX2dEa2NAcRPs3Pfchwua40ByY0g0DASgLybKJs11TLHqw?e=vxRjFa)
+
 #### setIdentity()
```

Comment:
Hi @sunnywu . For iOS, generateIdentity links to Client-Server guide and setIdentity links to Client-Side guide. Is this the wrong way around? I believe it's the opposite for Android.